test/plots/stackedAreaPlotTests.ts

```diff
@@ -396,7 +397,66 @@ describe("Plots", () => {
       assert.strictEqual(area1.attr("check"), "b", "projector has been applied to second area");
       svg.remove();
     });
+  });
+
+  describe("downsampling", () => {
+    it("renders correctly when downsampling is enabled", () => {
+      let svg = TestMethods.generateSVG(5, 10);
+      let xScale = new Plottable.Scales.Linear();
+      xScale.domain([0, 5]);
+      let yScale = new Plottable.Scales.Linear();
+      yScale.domain([0, 5]);
+      let data1 = [
+        { x: 1, y: 1}, // start point should not be removed
+        { x: 2, y: 1}, // points should be removed
+        { x: 3, y: 1}, // points should not be removed
+        { x: 4, y: 1}, // end point should not be removed
+      ];
+
+      let data2 = [
+        { x: 1, y: 1},
+        { x: 2, y: 2},
+        { x: 3, y: 3},
+        { x: 4, y: 2},
+      ];
 
+      let renderer = new Plottable.Plots.StackedArea<number>();
```

Comment:
let's not use the name `renderer`.  It is an artifact of really old history.  Let's just call it a plot.